Package.swift

```diff
@@ -33,10 +33,14 @@ let package = Package(
         .library(
             name: "SwaggerServiceModel",
             targets: ["SwaggerServiceModel"]),
+        .library(
+            name: "OpenAPIServiceModel",
+            targets: ["OpenAPIServiceModel"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/tachyonics/SwaggerParser.git", from: "0.6.4"), 
+        .package(url: "https://github.com/tachyonics/SwaggerParser.git", from: "0.6.4"),
         .package(url: "https://github.com/jpsim/Yams.git", from: "4.0.0"),
+        .package(url: "https://github.com/mndzup/OpenAPIKit.git", .branch("working")),
```

Comment:
nit: Make sure to switch this to the main branch before merging.

Reply:
Done.